scripts/deploy.ts

```diff
@@ -16,15 +16,27 @@ async function getConfig() {
 
   const argv = await yargs
     .env('')
+    .option('use-defender', {
+      description: 'Deploy using OpenZeppelin Defender',
+      type: 'boolean',
+      implies: ['deploy-salt'],
+    })
     .option('deploy-salt', {
       description: 'Salt to use for CREATE2 deployments',
       type: 'string',
-      demandOption: true,
     })
     .option('owner-address', {
       description: 'Address of the address to use as owner',
       type: 'string',
       demandOption: true,
+    })
+    .check((argv) => {
```

Comment:
Might be nice to ensure (below) that we only actually use Defender if `use-defender` is true.

Reply:
oops -- good catch